packages/ripple-binary-codec/test/tx-encode-decode.test.js

```diff
@@ -1,3 +1,4 @@
+const { expect } = require('chai')
```

Comment:
This shouldn't be here.

Reply:
Ah, I'm not sure why that ended up in there in the 1st place, but it was the root of the other issues I was running into. Reverting the changes to the test syntax.